frontend-react/src/pages/Validate.tsx

```diff
@@ -21,3 +27,11 @@ const Validate = () => {
 };
 
 export default Validate;
+
+export const ValidateWithAuth = () => (
+    <AuthElement
+        element={Validate}
+        requiredUserType={MemberType.PRIME_ADMIN}
+        requiredFeatureFlag={FeatureFlagName.VALIDATION_SERVICE}
```

Comment:
Can there be multiple flags?

Reply:
I considered it, but really we associate one flag to one route (like `/validate`, and `/value-sets`)